LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Oskar Hahn
+Copyright (c) since 2016 Oskar Hahn
```

Comment:
I think the year in this file marks the year of the constitution of the copyright for this intelectual property. This is relevant for the year when the copyright expires. So you do not have to change the file every year but only if you add some changes in a new year.

What about adding a small script to Travis CI that checks every pull request and checks if the current year is present?